STYLE_GUIDE.md

```diff
@@ -0,0 +1,156 @@
+# ComplyTime Organization Style Guide
+
+This document serves as the central source of truth for engineering standards, contribution workflows, and architectural principles for the ComplyTime organization. All contributors and maintainers are expected to adhere to these guidelines to ensure consistency, quality, and compliance.
+
+---
+
+## 1. High-Level Engineering Principles
+
+These principles guide *how* we think about problems and *why* we make specific technical decisions. We value long-term sustainability over short-term speed.
+
+### Core Value: Maintainability & Robustness
+
+**1. Single Source of Truth (Centralized Constants)**
+* **Principle:** Centralize values that are used in multiple places or may change over time.
+* **Implementation:**
+    * **No Magic Strings/Numbers:** Avoid using raw strings (e.g., `"active"`, `"https://api..."`) or raw numbers (e.g., `86400`) inline within your logic.
```

Comment:
Thinking if "Magic Strings/Numbers" wouldn't fit better in readability section.

Reply:
I think i can fit both. `Maintainability & Robustness` and `Readability`. Maybe we should combine these two sections into one, `Maintainability & Robustness & Readability`. WDYT